spec/features/user_signs_out_spec.rb

```diff
@@ -0,0 +1,15 @@
+require "rails_helper"
+
+feature "user can log out" do
+  scenario "presses logout" do
+    visit new_user_session_path
+    user = FactoryBot.create(:user)
+    
+    fill_in 'Email', with: user.email
+    fill_in 'Password', with: user.password
+    click_button 'Log in'
+    click_link 'Sign Out'
```

Comment:
Not necessary, but there's a lot of test duplication in this sign out spec having to do with signing the user in. Alternatively, you can use the Devise helper method `login_as` to sign in a user easily without filling out the form!